CHANGES.md

```diff
@@ -155,6 +155,8 @@ To be released.
     specific peer. [[#1240]]
  -  Added the option `--no-passphrase` to `planet key remove` command to remove
     key without asking passphrase.  [[#1213], [#1265]]
+ -  Added `planet key convert private-key` subcommand to convert private key
+    into address or public key. [[#1268]]
```

Comment:
How about using the term “derive”, which is conceptually more accurate, instead of “convert”?

```suggestion
 -  Added `planet key derive PRIVATE-KEY` subcommand to derive the address or
    public key from a private key.  [[#1268]]
```

Reply:
I applied your suggestion. Could you check it? The commits are a35c800, 05b04c3, 7517a10.